cnd/src/network/swarm.rs

```diff
@@ -152,6 +157,30 @@ impl Swarm {
         Ok(())
     }
 
+    pub async fn dial_peer(
+        &self,
+        peer_id: PeerId,
+        hint_addresses: Vec<Multiaddr>,
+    ) -> anyhow::Result<()> {
+        let mut swarm = self.inner.lock().await;
+
+        Self::dial_peer_on_swarm(&mut *swarm, peer_id, hint_addresses).await
+    }
```

Comment:
Is this one actually used? Could remove it and rename the other function to `dial_peer` if not :)